helm_deploy/values-dev.yaml

```diff
@@ -29,3 +29,4 @@ generic-prometheus-alerts:
   # Note: the topics dont have nice names, so just supply the name as a key, and a nice name as a value.
   snsAlertsTopicNames:
     cloud-platform-Digital-Prison-Services-f221e27fcfcf78f6ab4f4c3cc165eee7: "offender-events"
+  snsAlertsNoMessagesThreshold: 150
```

Comment:
We could rename `snsAlertsNoMessagesThreshold` to something like `snsAlertsNoMessagesThresholdMinutes` so it's clear for folks when they're configuring the value

Looking at https://github.com/ministryofjustice/hmpps-helm-charts/pull/37/files it has `no messages published in last {{ $snsAlertsNoMessagesThreshold }} mins` so it's clear there, but might be useful to make it clearer in each project